### PR TITLE
Fix some issues when install ioBroker on osx

### PR DESCRIPTION
**Pull request by SchumyHao**
When I tried to install ioBroker on osx. I found some issue#113 

I tried to fix those issues. This PR is tested OK on my mac.

### CODE REVIEW

**Comment by SchumyHao** [comment]
It seems AppVeyor test failed due to network issue. Not caused by this patch

**Comment by Apollon77** [comment]
Cool, thank you. 

Is „Brew“ always installed or do User need to install it before? Will it work with normal user to install the package?

**Comment by AlCalzone** [comment]
I assume you tested starting and stopping of the service? Does `iobroker start/stop/status` work correctly? With systemd, those commands were behaving weird, so I had to resort to using `controller.js` directly.

**Comment by SchumyHao** [comment]
@Apollon77 
brew need install before.
so we should update README.md am I right?

**Comment by SchumyHao** [comment]
@AlCalzone I have tested those command. You can find I use `launchctl load&unload` to start and stop iobroker

**Comment by AlCalzone** [comment]
@SchumyHao I saw those commands. It looks similar to what we did with systemd and that caused `iobroker status` to report the wrong status. If it works for you then I'm fine with it.

**Comment by Apollon77** [comment]
>brew need install before.
so we should update README.md am I right?

I see three options:
A) Update README and define as "needed for MacOS", but then theinstaller should also check for existence of brew and quit if not installed on macos with the info that it is needed and ideally a link where to get it :-)

B) Declare is as "optional/nice to have", means adding to README and recommended, installer checks for existence and if not existing just output a Info/Warnung to the user about "hey we wanted to install the following packages but were not able because brew not there, please install them manually" (not the words for the info, but the general "message behind it").

C) Check for existence and install brew if not installed ... But here I'm unsure if this is possible that easy without being admin/usung sudo or doing as root account. Me personally had lot's of fun on my last mac regarding installing brew and it ended up in chaos :-( But maybe it only was me.

I have installed git using other installers and not brew on my current mac ...

So if the packages are not 100% needed for installation I personally would tend to option B, but it is mostly because my personal "bad experience" with brew :-( 

I'm 100% open for other ideas and discussion.

**Comment by SchumyHao** [comment]
> @SchumyHao I saw those commands. It looks similar to what we did with systemd and that caused `iobroker status` to report the wrong status. If it works for you then I'm fine with it.

As far as I know, iobroker status works fine on osx.
If I start iobroker by command `iobroker start`, which will enable iobroker by load pfile. `iobroker status` shows *iobroker is running*
If i stop iobroker, `iobroker status` shows *iobroker is not running*

**Comment by Apollon77** [comment]
cool thank you!, also checking soon

File: installer.sh [hunk 1]
```diff
@@ -371,9 +372,9 @@ case "$platform" in
 				install_package_macos $pkg
 			done
 		else
-			echo "${yellow}Some dependency utils need installed though brew, which is not installed."
-			echo "If the installation is failed due to dependedncy utils not found."
-			echo "Please install it manually.${normal}"
+			echo "${yellow}Since brew is not installed, frequently-used dependencies could not be installed."
```
Comment (Apollon77) [inline installer.sh:375]: should we not output here also the "affected" packages? or we need them in doc and a link, or?

Comment (AlCalzone) [reply installer.sh:375]: I'm not sure, that seems like to much noise. That's why I referred to the adapter documentations.
But I'm fine with both.

Comment (AlCalzone) [reply installer.sh:375]: I'm not sure, that seems like to much noise. That's why I referred to the adapter documentations.
But I'm fine with both.

Comment (Apollon77) [reply installer.sh:375]: Ok, then we leave as is. If we get user feedback we can change it still afterwards

**Pull request merged**